WebApp/autoreduce_webapp/templates/help.html

```diff
@@ -1,17 +1,17 @@
 {% extends "base.html" %}
+{% block title %}Help pages{% endblock %}
 {% load static %}
 
 {% block stylesheets %}
-<link rel="stylesheet" href="{% static "css/help.css" %}">
+    <link rel="stylesheet" href="{% static "css/help.css" %}">
 
 {% endblock %}
 
 {% block body %}
-    <title>Help Pages</title>
     <div class="row">
         <div class="col-md-12 text-center">
             <h2>Help Pages</h2>
-            <p>Get in touch with the <a href="mailto:{{support_email}}">Support Team</a></p>
+            <p>Get in touch with the <a href="mailto:{{ support_email }}">Support Team</a></p>
         </div>
     </div>
     <!-- Temporarily disabled https://github.com/ISISScientificComputing/autoreduce/issues/701 -->
```

Comment:
Code in `help.js` references a search feature and so it might be confusing if that bit of code was removed completely. Should be added back soon anyway.